tools/tflitefile_tool/tests/test_setup.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os.path
+import unittest
+
+# Python doesn't have const var but handle these as const
+# It's meaning that DO NOT MODIFY these vars
+CACHED_TEST_MODEL_DIR = "../../tests/scripts/models/cache"
+TEST_MODEL_PATH = os.path.join(CACHED_TEST_MODEL_DIR, "convolution_test.tflite")
```

Comment:
If we want to use simple test model on host without download, we can use `nnpackage/examples/v1.0.0/add/add.tflite` file.

Reply:
If so, I'll change. I just need a simple tflite model.